core/stateful_precompile_test.go

```diff
@@ -581,6 +583,27 @@ func TestContractNativeMinterRun(t *testing.T) {
 				assert.Equal(t, common.Big1, state.GetBalance(allowAddr), "expected minted funds")
 			},
 		},
+		"initial mint funds": {
+			caller:         allowAddr,
+			precompileAddr: precompile.ContractNativeMinterAddress,
+			config: &precompile.ContractNativeMinterConfig{
+				InitialMint: map[common.Address]*math.HexOrDecimal256{
+					allowAddr: math.NewHexOrDecimal256(2),
+				},
+			},
+			input: func() []byte {
+				return precompile.PackReadAllowList(noRoleAddr)
+			},
+			suppliedGas: precompile.ReadAllowListGasCost,
+			readOnly:    false,
+			expectedRes: common.Hash(precompile.AllowListNoRole).Bytes(),
+			assertState: func(t *testing.T, state *state.StateDB) {
+				res := precompile.GetContractNativeMinterStatus(state, allowAddr)
+				assert.Equal(t, precompile.AllowListEnabled, res)
+
+				assert.Equal(t, common.Big2, state.GetBalance(allowAddr), "expected minted funds")
```

Comment:
I was confused by this address being required to be enabled here since it's not included in the config. Should we switch from performing some setup at the start of `t.Run` to always including the config in each of these tests so that it can configure the state as necessary?

Reply:
yup removed unneccessary ones